### PR TITLE
Call soffice with timeout

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: report_aeroo/report_aeroo.py [hunk 1]
```diff
@@ -104,14 +104,51 @@ def create_aeroo_report(
 
             filedir, filename = os.path.split(temp_file.name)
 
-            subprocess.check_output([
+            proc = subprocess.Popen([
```
Comment (joaoalf) [inline report_aeroo/report_aeroo.py]: @dufresnedavid It would be nice to test the timeout

Comment (ghost) [reply report_aeroo/report_aeroo.py]: @joaoalf how would you test that

Comment (ghost) [reply report_aeroo/report_aeroo.py]: @joaoalf ok, I have an idea

**Comment by ghost** [comment]
@joaoalf Fixed

File: report_aeroo/report_aeroo.py (hunk 1 above)
Comment (joaoalf) [reply report_aeroo/report_aeroo.py]: @dufresnedavid I would save the executable name in a parameter to be able to call a stub script during tests.

**Comment by ghost** [comment]
@joaoalf ok, fixed again

**Comment by ghost** [comment]
@joaoalf can I merge ?

**Pull request merged**